### PR TITLE
Make path optional

### PR DESCRIPTION
**Pull request by mdesmet**
### Problem

Packages folder is not downloaded on dbt cloud, so we can't resolve the path to any dbt entities located there.

### Solution

Make path optional

### Screenshot/Demo

A picture is worth a thousand words. Please highlight the changes if applicable.

### How to test

- Steps to be followed to verify the solution or code changes
- Mention if there is any settings configuration added/changed/deleted

## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

**Comment by mdesmet** [comment]
@AdiGajbhiye , @saravmajestic: Can we assess the impact of making this change. What will be the effect on documentation editor, project health check and lineage.

**Comment by saravmajestic** [comment]
> @AdiGajbhiye , @saravmajestic: Can we assess the impact of making this change. What will be the effect on documentation editor, project health check and lineage.

just by looking at the changes, tests added as macros will be affected. I will test the doc editor and will let you know

File: src/services/dbtTestService.ts [hunk 1]
```diff
@@ -103,10 +103,18 @@ export class DbtTestService {
       return;
     }
 
-    const patchPath = node?.patch_path.includes("://")
+    const patchPath = node?.patch_path?.includes("://")
       ? path.join(project.projectRoot.fsPath, node.patch_path.split("://")[1])
       : node.patch_path;
 
+    if (!patchPath) {
```
Comment (mdesmet) [inline src/services/dbtTestService.ts:110]: This seems unrelated to this change. The patch_path will always be there, if there is no file exists check it should always exist.

Comment (saravmajestic) [reply src/services/dbtTestService.ts:110]: and this is the file exist check
